test-util/src/main/java/io/zeebe/test/util/asserts/TopologyAssert.java

```diff
@@ -44,6 +47,25 @@ public final TopologyAssert isComplete(final int clusterSize, final int partitio
           partitionCount, brokersWithUnexpectedPartitionCount);
     }
 
+    final Set<Integer> partitionsWithLeader =
+        brokers.stream()
+            .flatMap(b -> b.getPartitions().stream())
+            .filter(PartitionInfo::isLeader)
+            .map(PartitionInfo::getPartitionId)
+            .collect(Collectors.toUnmodifiableSet());
+    final Set<Integer> partitionsWithoutLeader =
+        brokers.stream()
+            .flatMap(b -> b.getPartitions().stream())
+            .filter(PartitionInfo::isLeader)
```

Comment:
```suggestion
            .filter(Predicate.not(PartitionInfo::isLeader))
```

Reply:
:see_no_evil:

Reply:
Actually the line had to be removed - what I want is set difference. Actually, now that I think about it, this is a very round about way of doing it. Let me check if I can't make it simpler.